store-and-retrieve-data/use-storagehub-sdk/get-started.md

```diff
@@ -78,6 +78,139 @@ Add the core and MSP client packages to your project. These libraries provide th
     npm install @storagehub-sdk/core @storagehub-sdk/msp-client
     ```
 
+## Initialize the StorageHub SDK
+
+In this step, you will learn how to initialize all the clients you will need while building with the StorageHub SDK. You will go through installing extra dependencies, setting up `walletClient`, `publicClient`, `storageHubClient`, `polkadotApi`, `mspClient`, and a few helper functions related to the `mspClient` to help get you started with things like checking MSP health and user authentication.
+
+### Install Additional Dependencies
+
+=== "pnpm"
+
+    ```bash { .break-spaces }
+    pnpm add @storagehub/types-bundle @polkadot/api @storagehub/api-augment viem
+    ```
+
+=== "yarn"
+
+    ```bash { .break-spaces }
+    yarn add @storagehub/types-bundle @polkadot/api @storagehub/api-augment viem
+    ```
+
+=== "npm"
+
+    ```bash { .break-spaces }
+    npm install @storagehub/types-bundle @polkadot/api @storagehub/api-augment viem
+    ```
+
+??? interface "Why do I need these dependencies?"
+
+    - **[`@storagehub/types-bundle`](https://www.npmjs.com/package/@storagehub/types-bundle){target=_blank}:** Describes DataHaven's custom on-chain types.
+
+    - **[`@polkadot/api`](https://www.npmjs.com/package/@polkadot/api){target=_blank}:** The core JavaScript library used to talk to any Substrate-based blockchain, which in our case is DataHaven.
+
+    - **[`@storagehub/api-augment`](https://www.npmjs.com/package/@storagehub/api-augment){target=_blank}:** Extends `@polkadot/api` with DataHaven's custom pallets and RPC methods. You will import it in your `index.ts` file where your main script logic will be executed.
+
+    - **[`viem`](https://www.npmjs.com/package/viem){target=_blank}:** Lightweight library for building Ethereum-compatible applications.
+
+### Set Up Client Service
+
+1. Within your `src` folder where `index.ts` should be located, create a new folder called `services` as follows:
+
+    ```shell
+    mkdir services && cd services
+    ```
+
+    You'll need to set up the necessary clients to connect to the DataHaven network, which runs on a dual-protocol architecture (Substrate for core logic and EVM for compatibility).
```

Comment:
Ah ok, so this sentence might be more useful as the intro sentence before you dive into the steps